pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -448,6 +451,16 @@ type JaegerCassandraCreateSchemaSpec struct {
 	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty"`
 }
 
+// GRPCPluginSpec represents the GRPC Plugin init container
+// +k8s:openapi-gen=true
+type GRPCPluginSpec struct {
+	Image string `json:"image,omitempty"`
```

Comment:
Re. required.

for binary, and probably config, there can be a default value like /plugin/start and 
/etc/plugin-config/config.json, but as we leave it up to the user to provide the volume, instead of hard coding it as /plugin, I think this is a slightly strange mix of implicit and explicit.

Reply:
Added comments. Made config optional.

When I wrote my plugin it was a bit confusing that the options to jaeger is configuration-file and the option jaeger starts the plugin with is --config. I documented this but it might be too verbose..